### ISSUE TITLE
Automatically update apps

### ISSUE THREAD

**Issue by epiepo** [sandstorm-io/sandstorm#2875]
**Feature Request**

This issue is NOT meant as a dupe of [Issue #208](https://github.com/sandstorm-io/sandstorm/issues/208), so bear with me please. 

At current, users get notifications about their apps needing updates. However, I don't want my users to be bothered by this, I want the apps to update automatically in the background. Please provide an option for this in the Admin Panel.

**Comment by ocdtrekkie** [comment]
I know this is something they want to implement, but I am not sure the admin panel is the right place for that. Because most users will indeed want apps to update automatically, but some users will not. This should be a user-level control, not a server-level control.

There are several planned changes to how updates work on the roadmap, which can be found here: https://github.com/sandstorm-io/sandstorm/tree/master/roadmap/platform/apps#updates

**Comment by kentonv** [comment]
Yes, I want to improve this. Another problem with non-automated updates is that it means that servers with many users tend to get stuck storing every version of every app, since there's always someone who hasn't updated. But we do need to make sure users have recourse when they receive a broken update...
